modules/nf-core/ampcombi2/parsetables/main.nf

```diff
@@ -4,43 +4,52 @@ process AMPCOMBI2_PARSETABLES {
 
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/ampcombi:0.2.2--pyhdfd78af_0':
-        'biocontainers/ampcombi:0.2.2--pyhdfd78af_0' }"
+        'https://depot.galaxyproject.org/singularity/ampcombi:2.0.1--pyhdfd78af_0':
+        'biocontainers/ampcombi:2.0.1--pyhdfd78af_0' }"
 
     input:
     tuple val(meta), path(amp_input)
     path(faa_input)
     path(gbk_input)
-    path(opt_amp_db)
+    val(opt_amp_db)
+    path(opt_amp_db_dir)
 
     output:
-    tuple val(meta), path("${meta.id}/")                              , emit: sample_dir
-    tuple val(meta), path("${meta.id}/contig_gbks/")                  , emit: contig_gbks
-    tuple val(meta), path("${meta.id}/${meta.id}_diamond_matches.txt"), emit: txt
-    tuple val(meta), path("${meta.id}/${meta.id}_ampcombi.tsv")       , emit: tsv
-    tuple val(meta), path("${meta.id}/${meta.id}_amp.faa")            , emit: faa
-    tuple val(meta), path("${meta.id}/${meta.id}_ampcombi.log")       , emit: sample_log, optional:true
-    tuple val(meta), path("Ampcombi_parse_tables.log")                , emit: full_log, optional:true
-    tuple val(meta), path("amp_ref_database/")                        , emit: results_db, optional:true
-    tuple val(meta), path("amp_ref_database/*.dmnd")                  , emit: results_db_dmnd, optional:true
-    tuple val(meta), path("amp_ref_database/*.clean.fasta")           , emit: results_db_fasta, optional:true
-    tuple val(meta), path("amp_ref_database/*.tsv")                   , emit: results_db_tsv, optional:true
-    path "versions.yml"                                               , emit: versions
+    tuple val(meta), path("${meta.id}/")                                        , emit: sample_dir
+    tuple val(meta), path("${meta.id}/contig_gbks/")                            , emit: contig_gbks
+    tuple val(meta), path("${meta.id}/${meta.id}_mmseqs_matches.tsv")           , emit: db_tsv
+    tuple val(meta), path("${meta.id}/${meta.id}_ampcombi.tsv")                 , emit: tsv
+    tuple val(meta), path("${meta.id}/${meta.id}_amp.faa")                      , emit: faa
+    tuple val(meta), path("${meta.id}/${meta.id}_ampcombi.log")                 , emit: sample_log, optional:true
+    tuple val(meta), path("Ampcombi_parse_tables.log")                          , emit: full_log, optional:true
+    tuple val(meta), path("amp_${opt_amp_db}_database/")                        , emit: db, optional:true
```

Comment:
Ooof, thats alot... does it make sense to export every single file within mmseqs as a separate file? I would assume only the directory as a whole could be consumed downstream, or?

Reply:
Also don't forget to format with the NExtflow formatter if you've not already, I think this is one of the parts of the harshil alignment not yet implemented

Reply:
well the files generated are not mush. Im not sure what the standard here, because sometimes we emit these db files and sometimes we dont.